final_prep/README.md

```diff
@@ -32,24 +32,25 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+[calendar](/Users/jessicajones/Desktop/W1.png)
+[calendar](Users/jessicajones/Desktop/W2.png)
+[calendar](Users/jessicajones/Desktop/W3.png)
```

Comment:
I do not see screenshots of your calendar. This is formatted like a link in markdown. Perhaps just putting a `!` at the front will help these show up. Remember to bookmark the [markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)!